modules/patient-portal/src/main/frontend/src/patient-portal/PatientAccessConfiguration.jsx

```diff
@@ -53,22 +55,41 @@ function PatientAccessConfiguration() {
 
   const [ patientAccessConfig, setPatientAccessConfig ] = useState();
   const [ hasChanges, setHasChanges ] = useState(false);
+  const [ draftLifetime, setDraftLifetime ] = useState();
+  const [ error, setError ] = useState(false);
+  const inputRef = useRef(null);
 
+  // Boolean fields can have one label
+  // Text fields can have one label and one optional helper text
   const labels = {
     tokenlessAuthEnabled: "Patients can answer surveys without a personalized link",
     PIIAuthRequired: "Patients must confirm their identity by providing their date of birth and either MRN or HCN",
-    allowedPostVisitCompletionTime: "Patients can fill out surveys after the associated event for:"
+    allowedPostVisitCompletionTime: ["Patients can fill out surveys after the associated event for:"],
+    draftLifetime: ["Patients can edit unsubmitted responses for:", "-1 means that drafts are kept until the patient is no longer able to access their surveys"]
   };
+  const errorText = `Value must be between ${DEFAULT_PATIENT_ACCESS_CONFIG['draftLifetime']} and ${patientAccessConfig?.allowedPostVisitCompletionTime}`;
+
+  let readpatientAccessConfigData = (json) => {
```

Comment:
Please fix inconsistent casing:

```suggestion
  let readPatientAccessConfigData = (json) => {
```